### PR TITLE
0.1/604 options screen menu

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: app/Http/Controllers/Service/DashboardController.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+    /**
+     * Index the Dashboard options
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $data = [
+            "user_name" => $user->name,
+            "centre_name" => $user->centre->name,
```
Comment (katjam) [inline app/Http/Controllers/Service/DashboardController.php]: centre_id is nullable. Do we need to make sure this user has one?

File: app/Http/Middleware/RouteAutoAuthenticate.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @param  string|null  $guard
+     * @return mixed
+     */
+    public function handle($request, Closure $next, $guard = null)
+    {
+        // check request for a user param; use #1 if there isn't one.
+        $user_id = ($request->query('user')) ? $request->query('user') : 1;
```
Comment (katjam) [inline app/Http/Middleware/RouteAutoAuthenticate.php:21]: Who is user one? Do we want to hard code that in?

File: app/User.php [hunk 3]
```diff
@@ -27,8 +27,14 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
+
```
Comment (katjam) [inline app/User.php]: Extra space.

File: app/User.php [hunk 4]
```diff
@@ -27,8 +27,14 @@ class User extends Authenticatable
...
     public function notes()
     {
         return $this->hasMany('App\Note');
     }
+
+    public function centre()
```
Comment (katjam) [inline app/User.php]: Boring of me to say - but missing doc comments.

**Review by katjam — COMMENTED** [review]
Pretty close.

File: app/Http/Controllers/Service/DashboardController.php (hunk 1 above)
Comment (charlesstrange2) [reply app/Http/Controllers/Service/DashboardController.php]: We do, I'll change it.

File: app/Http/Middleware/RouteAutoAuthenticate.php (hunk 2 above)
Comment (charlesstrange2) [reply app/Http/Middleware/RouteAutoAuthenticate.php:21]: User one is setup by the supporting seeds. System will throw an exception if user one doesn't exist.
At this point i just want a default user with the option to fiddle the url to find another one.

File: app/User.php (hunk 3 above)
Comment (charlesstrange2) [reply app/User.php]: will remove.

File: app/User.php (hunk 4 above)
Comment (charlesstrange2) [reply app/User.php]: okedoke.

**Review by katjam — APPROVED** [review]
Looks good to me! :lollipop:

**Pull request merged**